addons/account/views/account_analytic_distribution_model_views.xml

```diff
@@ -25,7 +26,8 @@
             <field name="arch" type="xml">
                 <data>
                     <xpath expr="//field[@name='partner_category_id']" position="after">
-                            <field name="account_prefix"/>
+                        <field name="prefix_placeholder" invisible="1"/>
```

Comment:
I know invisible field are not needed anymore but here we need it for the widget

Reply:
I think you need to put a comment on the line to justify it, so the test won't pick it up.
(the test is deactivated currently, but will be put in application at some point)